src/libraries/System.Runtime.Serialization.Xml/tests/XsdDataContractExporterTests/SurrogateTests.cs

```diff
@@ -69,7 +69,7 @@ public static IEnumerable<object[]> SurrogateProvider_MemberData()
                 SchemaUtils.OrderedContains(@"<xs:complexType name=""SurrogateTests.SerializableNode"">", ref s);
                 SchemaUtils.OrderedContains(@"<Surrogate xmlns:i=""http://www.w3.org/2001/XMLSchema-instance"" z:Id=""1"" xmlns:d1p1=""http://schemas.datacontract.org/2004/07/System"" i:type=""d1p1:Version"" xmlns:z=""http://schemas.microsoft.com/2003/10/Serialization/"" xmlns=""http://schemas.microsoft.com/2003/10/Serialization/"">", ref s);
                 SchemaUtils.OrderedContains(@"<d1p1:_Build>0</d1p1:_Build>", ref s);
-                SchemaUtils.OrderedContains(@"<d1p1:_Major>7</d1p1:_Major>", ref s);
+                SchemaUtils.OrderedContains(@"<d1p1:_Major>8</d1p1:_Major>", ref s);
```

Comment:
Can you avoid the hardcoded '8' by using your approach from L58?

Reply:
Yes. Good catch.